.github/workflows/docs.yml

```
@@ -40,7 +40,8 @@ jobs:
       - uses: actions/checkout@v4
       
       - name: Rust toolchain
-        uses: dtolnay/rust-toolchain@stable
+        name: Install pinned Rust toolchain
```

actionlint (1.7.12)

[error] key "name" is duplicated in element of "steps" section. previously defined at line:42,col:9

(syntax-check)